_posts/0000-01-02-ruthielin.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"The first step is to establish that something is possible; then probability will occur."